internal/repos/sync_worker.go

```diff
@@ -74,7 +74,7 @@ func NewSyncWorker(ctx context.Context, logger log.Logger, dbHandle basestore.Tr
 		Metrics:           newWorkerMetrics(opts.PrometheusRegisterer),
 	})
 
-	resetter := dbworker.NewResetter(store, dbworker.ResetterOptions{
+	resetter := dbworker.NewResetter(logger.Scoped("dbworker.Resetter", ""), store, dbworker.ResetterOptions{
```

Comment:
```suggestion
	resetter := dbworker.NewResetter(logger.Scoped("repo.sync.worker.Resetter", ""), store, dbworker.ResetterOptions{
```
Based on the name below ?